docs/vcr_recording.md

```diff
@@ -0,0 +1,60 @@
+# VCR Recording and Rerecording
+Occassionally it is important to update VCR cassettes. This ensures that tests
+which rely on external service stubs from VCR are not returning false positives.
+
+## Setup
+
+### Benefits checker
+ - You will need to set the following env vars in `.env.test` to use the staging environments details.
+
+  ```
+  BC_LSC_SERVICE_NAME=
+  BC_CLIENT_ORG_ID=
+  BC_CLIENT_USER_ID=
+  BC_WSDL_URL=
+  BC_USE_DEV_MOCK=false
+  ```
+ - Important: turn off the benefit checker mock (`BC_USE_DEV_MOCK=false`) when recording, but switch back on later (does it matter?)
```

Comment:
might be good to say, 'switch BC_USE_DEV_MOCK to false, if it is enabled, and use the staging benefit checker'. e.g. I always use the mock service locally but you could just as easily use the staging benefit checker and never use the mock service

Reply:
have tweaked that now @willc-work